lib/error-handler.js

```diff
@@ -40,7 +40,8 @@ module.exports = settings => {
         Component: Component.default || Component,
         scripts: [],
         url,
-        error
+        error,
+        isAsruUser: !!req.user.profile.asruUser
```

Comment:
This should be replaced with a `lodash#get` because there's no guarantee that `req.user.profile` will exist in an error state, and error pages that throw more errors is a bad look.